src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/DefendantEntity.java

```diff
@@ -42,11 +43,16 @@
     @JsonIgnore
     private final Long id;
 
+    @ToString.Exclude
     @ManyToOne(optional = false)
     @JoinColumn(name = "COURT_CASE_ID", referencedColumnName = "id", nullable = false)
     @Setter
     private CourtCaseEntity courtCase;
 
+    @Column(name = "UUID", nullable = false)
+    private final String uuid;
```

Comment:
I'm torn as to whether `uuid` is a good name for this 🤔  I appreciate it is one but it feels wrong using the name of a data type where we probably ought to have a business concept. For example you wouldn't call a field `longId` or `stringId`. That said its business purpose is to provide a universally unique identifier... so maybe `uuid` *is* appropriate?

Reply:
Yes - I thought the same as I was doing it. We use `caseId` in the `courtCaseEntity` to store the UUID, albeit that was partly because we were reusing an existing field. So maybe we should say `defendantId` ? The fact you mentioned it too means maybe we should do something.